csharp/src/Ice/TcpEndpoint.cs

```diff
@@ -228,19 +226,15 @@ protected internal override void WriteOptions(OutputStream ostr)
         private protected override IConnector CreateConnector(EndPoint addr, INetworkProxy? proxy) =>
             new TcpConnector(this, addr, proxy);
 
-        private protected override IPEndpoint CreateIPEndpoint(
-            string host,
-            ushort port,
-            bool compressionFlag,
-            TimeSpan timeout) =>
+        private protected override IPEndpoint CloneWithHostAndPort(string host, ushort port) =>
```

Comment:
Should Clone return this if host == Host and port == Port?

Reply:
Yes. I don't see why not